antispam_bee.php

```diff
@@ -2391,6 +2408,21 @@ public static function delete_spam_reason_by_comment( $comment_id ) {
 		);
 	}
 
+	/**
+	 * Updates the Antispam Bee reason for manual transitions
+	 *
+	 * @since   2.9.2
+	 * @param  [int] $comment Comment Object or comment ID.
```

Comment:
Isn't this a WP_Comment object? 
https://github.com/WordPress/WordPress/blob/8f1eddef1176dc495ff5e9043617685adf16c62e/wp-includes/comment.php#L1730

Reply:
Yes, you are right. I've copied the code from a Stackexchange function, but this function needed to cover `comment_post` additionally for example which is using a comment ID. Therefore the test if it is an object. I will remove it.